core/src/main/scala/runtime/metrics.scala

```diff
@@ -22,41 +22,38 @@ package metrics
 import freestyle._
 import freestyle.implicits._
 import freestyle.fs2.implicits._
-import freestyle.opscenter.model.Metric
+import freestyle.opscenter.model.{ListMetrics, Metric}
 import freestyle.opscenter.runtime.metrics.implicits._
-import cats.implicits._
-import cats.Applicative
 import _root_.fs2.{time, Scheduler, Strategy, Stream, Task}
 import _root_.fs2._
-import org.http4s.websocket.WebsocketBits.Text
+import org.http4s.websocket.WebsocketBits.{Binary, Text}
 import org.http4s.websocket.WebsocketBits.WebSocketFrame
 import java.time.Instant
+
 import scala.concurrent.duration._
 import scala.util.Random
 
 object implicits {
 
-  import cats.syntax.applicative._
-
   implicit val scheduler = Scheduler.fromFixedDaemonPool(2)
   implicit val strategy =
     Strategy.fromExecutionContext(scala.concurrent.ExecutionContext.Implicits.global)
 
-  implicit def metricsHandler[M[_]: Applicative](implicit C: Capture[M]): MetricsM.Handler[M] =
+  implicit def metricsHandler[M[_]](implicit C: Capture[M]): MetricsM.Handler[M] =
     new MetricsM.Handler[M] {
 
-      private def randomMetrics: List[Metric[Int]] = {
-        val microservices = List("analytics", "users", "payments")
-        val nodes         = List("node-1", "node-2", "node-4")
-        val metrics       = List("cassandra.queue", "instance.cpu.usage", "instance.cpu.disk")
+      private def randomMetrics: Seq[Metric] = {
+        val microservices = Seq("analytics", "users", "payments")
+        val nodes         = Seq("node-1", "node-2", "node-4")
+        val metrics       = Seq("cassandra.queue", "instance.cpu.usage", "instance.cpu.disk")
 
         for {
           microservice <- microservices
           node         <- nodes
           metric       <- metrics
           value     = Random.nextInt()
           timestamp = Instant.now.getEpochSecond
-        } yield Metric[Int](metric, microservice, node, value, timestamp)
+        } yield Metric(metric, microservice, node, value.toFloat, timestamp)
```

Comment:
It can be addressed in a different PR, but regarding the `timestamp`,  let's use JodaTime -> `UTC`

Reply:
Created issue for this https://github.com/frees-io/freestyle-opscenter/issues/15